src/helpers/analytics.ts

```diff
@@ -75,6 +76,15 @@ const analytics = {
 export const registerGlobalAnalytics = (window: Window, store: Store) => {
   const document = window.document;
 
+  // google optimize - will dispatch a feature flag on variant receieved
+  window.gtag('event', 'optimize.callback', {
+    callback: (idx: string, id: string) => {
+      store.dispatch(receiveExperiments([id, idx]));
+    },
+  });
```

Comment:
@TomWoodward the test for this function fails on the exception `TypeError: window.gtag is not a function` because `gtag()` is defined in the head html. Is there a way I can mock it?

Reply:
you could throw it in `src/test/setup.ts` but to please typescript in either the tests or the regular app you'll need to add its type to `src/typings/global.d.ts`